open-machine-learning-jupyter-book/assignments/ml-fundamentals/build-classification-models.ipynb

```diff
@@ -5,13 +5,1070 @@
    "cell_type": "markdown",
```

Comment:
<p>The title should be more specific.</p><p> </p><p>Also, rename this file, it should be more specific as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>